client/testutil/driver_compatible.go

```diff
@@ -18,6 +18,12 @@ func QemuCompatible(t *testing.T) {
 	}
 }
 
+func RktCompatible(t *testing.T) {
+        if runtime.GOOS != "windows" && syscall.Geteuid() != 0 {
```

Comment:
This check might be off. Shouldn't we check if we both aren't on windows, and that we are root? So like, `GOOS == windows || Geteuid() != 0` ?